proposed/ObjectToObjects.md

```diff
@@ -0,0 +1,56 @@
+Object To Objects
+=================
+
+Summary
+-------
+
+Objects are always passed by reference, not by value. This means you always have shared state when passing objects around.
+Shared state is one of the main culprits for unnecessary or incidental complexity. Complexity is bad.
+
+re-But...tals
+-------------
+
+- "But I use value objects!" Good for you, that is not actually OOP though.
+
+- "But all my objects are immutable." Yup, same as value objects. See above.
+
+- "OOP is the future!". Carnegie Melloin disagrees, they don't actually offer OOP as a course anymore. OO Design does 
+  have merit. But you should probably check out this Functional Programming thing.
+
+In the wild
+-----------
+
+- DateTime, part of PHP. There is now also an ImmutableDateTime. Guess why.
+
+- PSR-7: URL class ->withMethod('OPTIONS') - this is good btw.
+
+Elaboration
+-----------
+
+Complexity is the main problem with software development. It is always better to keep things simple. Simple is not easy.
```

Comment:
I think that "Simple is not easy" and thoughts about complexity deserve a separate Fight.
In the very least it will give us something to link to.
That does not necessarily mean this should happen _now_, but it should be on our radar.